portal/apps/org.wso2.carbon.iam.portal.common/src/main/lang/en_US.properties

```diff
@@ -0,0 +1,14 @@
+profile.image.updated=Profile image successfully updated
+profile=Profile
+users.add.password=Password
+users.confirm.password=Confirm Password
+update.header=Update User
+user.profile.update.error=Error occured when updating user profile
+save=Save
+update=Update
+profile.success=Successfully updated the profile!
+profile.failure=An Error occured when updating the profile!
+profile.updated=Your profile is updated.
+invalid.profile.name=Invalid profile name.
+profile.image.loaded=Profile image is loaded
+account.locked=Account is locked.
```

Comment:
Add newline.

Reply:
Fix with https://github.com/wso2/product-is/pull/1372